test/gold/tools/getTrans/getTrans_articulate.txt

```diff
@@ -37,7 +37,7 @@ params.size: 0
         "subtraces": 0,
         "traceAddress": null,
         "type": "call",
-        "error": "invalid jump destination",
```

Comment:
This is very annoying. Erigon changed their error message in the latest release. One of my machines (linux) has the updated Erigon. The other (my laptop) does not.

Reply:
I've seen you change the tests in your PR. I guess this will fix itself when we merge

Reply:
We have to update our endpoints. That's easy for me, but now that I think about it, it may not be all that easy for you on dAppNode. (This is another reason for us to have our own package.)